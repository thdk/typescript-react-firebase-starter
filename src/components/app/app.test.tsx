import * as React from "react";

import { render, cleanup } from "@testing-library/react";
import { App } from "./";

afterEach(cleanup);

it("has one child", () => {
    const { container } = render(<App />);

    expect(container.children.length).toBe(1);
});

it("renders", () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
});
