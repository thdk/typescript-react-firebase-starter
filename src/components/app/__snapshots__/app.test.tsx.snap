// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <h1>
    Hello from 
    <span
      class="red"
    >
      Typescript
    </span>
     and 
    <span
      class="green"
    >
      React
    </span>
    !
  </h1>
</DocumentFragment>
`;
